docs/source/user_guide/command_line.rst

```diff
@@ -166,8 +168,8 @@ This will generate several output files:
 - The structure trajectory every 100 steps, written to ``NaCl-npt-T300.0-p1.0-traj.extxyz``
 - The structure to be able to restart the dynamics every 1000 steps, written to ``NaCl-npt-T300.0-p1.0-res-1000.extxyz``
 - The final structure written to ``NaCl-npt-T300.0-p1.0-final.extxyz``
-- A log of the processes carried out, written to ``md.log``
-- A summary of the inputs and start/end time, written to ``md_summary.yml``.
+- A log of the processes carried out, written to ``NaCl-npt-T300.0-p1.0-log.yml``
```

Comment:
here I will not add md, since the ensemble makes it pointless

Reply:
Unless you actively object to it, I've kept `md` in because for the log, we determine the name in `BaseCalculation`, so having different logic for md makes things more complicated than I think they need to be.